examples/basics/scene/stereo.py

```diff
@@ -8,6 +9,8 @@
 Example demonstrating stereo vision in a scene has an anisotropic aspect
 ratio. This example can be used to test that the cameras behave
 correctly with nested translated/rotated cameras.
+
+NOTE: This example is currently broken!
```

Comment:
I assume it is still possible though? Can we have this example fixed in this PR, since it's a good validation?